packages/hsl-shared-components/src/ActionList/ActionListNative.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+import { ListView } from 'react-native';
```

Comment:
This is also available via styled-components/native.
```
import styled from 'style-components/native';
...
const ListView = styled.ListView``;
```
According to react-native docs listview supports same props as scrollview and scrollview supports borders.